src/pvrclient-nextpvr.cpp

```diff
@@ -720,12 +726,25 @@ void cPVRClientNextPVR::PauseStream(bool bPaused)
   if (IsServerStreaming())
   {
     if (m_nowPlaying == Recording)
-      m_recordingBuffer->PauseStream(bPaused);
+      m_multistreamRecording[m_streamCount]->PauseStream(bPaused);
     else
       m_livePlayer->PauseStream(bPaused);
   }
 }
 
+
+PVR_ERROR cPVRClientNextPVR::PauseRecordedStream(int64_t streamId, bool bPaused)
+{
+  if (IsServerStreaming())
+  {
+    if (m_nowPlaying == Recording)
+      m_multistreamRecording[streamId]->PauseStream(bPaused);
```

Comment:
`std::lock_guard<std::recursive_mutex> lock(m_multistreamRecording);` prior to accessing map.

No need to stop the lock, happens automatically at the end block